mappings/net/minecraft/server/network/ServerPlayNetworkHandler.mapping

```diff
@@ -56,7 +56,13 @@ CLASS net/minecraft/class_3244 net/minecraft/server/network/ServerPlayNetworkHan
 		ARG 2 listener
 	METHOD method_14370 executeCommand (Ljava/lang/String;)V
 		ARG 1 input
-	METHOD method_14371 validateVehicleMove (DDDFF)Z
+	METHOD method_14371 isMovementInvalid (DDDFF)Z
+		COMMENT {@return whether this movement is invalid}
+		COMMENT
+		COMMENT @implNote This method is used to determine whether players sending
+		COMMENT sending {@linkplain PlayerMoveC2SPacket player} and
```

Comment:
`sending` twice in a row looks like a typo.

Reply:
Fixed in 3f862a6868a119dc6053e3f1fb71b3fb7573d166